src/AzureAppConfigurationOptions.ts

```diff
@@ -35,4 +36,8 @@ export interface AzureAppConfigurationOptions {
      * Specifies options used to resolve Vey Vault references.
      */
     keyVaultOptions?: AzureAppConfigurationKeyVaultOptions;
+    /**
+     * Specifies options for dynamic refresh key-values.
+     */
+    refreshOptions?: RefreshOptions;

```

Comment:
I know we finalized the name `RefreshOptions` because `AzureAppConfigurationOptions` already contains the prefix `AzureAppConfiguration`. 
Looking at other names in this class, I'm wondering if we should change those to drop the app config prefix too?
- `AzureAppConfigurationKeyVaultOptions` -> `KeyVaultOptions`
- `AppConfigurationClientOptions` -> `ConfigurationClientOptions`

Reply:
Good point. I agree.

Reply:
To make it clear, I'll create a separate PR updating it later.

Reply:
created #44